torchax/torchax/tensor.py

```diff
@@ -394,12 +394,9 @@ def _to_copy(self, the_tensor, new_dtype, new_device):
       
 
     def get_and_rotate_prng_key(self, generator: Optional[torch.Generator]=None):
-      # Always use the default `randint` to get the next seed
-      with mode_utils.no_dispatch(), torch._C.DisableTorchFunction():
-        next_key = torch.randint(
-            0, 2**32, (), dtype=torch.uint32, generator=generator).numpy()
+      self._prng_key, next_key = jax.random.split(self._prng_key)
```

Comment:
I think some tests failed because this didn't initialize `self._prng_key`

Related, WDYT introducing a `reset_prng_key` method so that the dynamo backend doesn't have to poke into the innards of this class?

Reply:
made this into a public field.